package.json

```diff
@@ -1,19 +1,19 @@
 {
```

Comment:
We should revert this file to its original contents if you want to merge these changes into this repo

Reply:
@vocamen ^^

Reply:
Go for it.
Maybe just mention vocamen contribution, what do you think?

Reply:
package.json can have a contributors field: https://nodejs.dev/learn/the-package-json-guide#contributors

Reply:
done!